src/components/Page/Page.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import Title from "../Title/Title";
+import CardContainer from '../CardContainer/CardContainer';
+
+class Page extends React.Component {
```

Comment:
As this doesn't use state at all (yet, anyway) we can use a stateless component https://reactjs.org/docs/components-and-props.html#function-and-class-components

Reply:
on it